components/infobox/wikis/valorant/infobox_person_player_custom.lua

```diff
@@ -85,7 +91,7 @@ function CustomInjector:parse(id, widgets)
 		return {
 			Cell{name = 'Status', content = CustomPlayer._getStatusContents(args)},
 			Cell{name = 'Years Active (Player)', content = {args.years_active}},
-			Cell{name = 'Years Active (Org)', content = {args.years_active_manage}},
+			Cell{name = 'Years Active (<abbr title="Organisation">Org</abbr>)', content = {args.years_active_org}},
```

Comment:
is `years_active_manage` currently used anywhere?
if so after this pr is merged you would need to adjust the usages accordingly